services/task-svc/internal/patient/patient.go

```diff
@@ -0,0 +1,103 @@
+package patient
+
+import (
+	"context"
+	"github.com/google/uuid"
+	zlog "github.com/rs/zerolog/log"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"hwgorm"
+	"task-svc/api"
+)
+
+type Base struct {
+	HumanReadableIdentifier string `gorm:"column:human_readable_identifier"`
+}
+
+type Patient struct {
+	Base
+	Id uuid.UUID `gorm:"column:id"`
+}
+
+type ServiceServer struct {
+	api.UnimplementedPatientServiceServer
+}
+
+func NewServiceServer() *ServiceServer {
+	return &ServiceServer{}
+}
+
+func (ServiceServer) CreatePatient(ctx context.Context, req *api.CreatePatientRequest) (*api.CreatePatientResponse, error) {
+	log := zlog.Ctx(ctx)
+	db := hwgorm.GetDB(ctx)
+
+	// TODO: Auth
+
+	patient := Patient{
+		Base: Base{
+			HumanReadableIdentifier: req.HumanReadableIdentifier,
+		},
+	}
+
+	if err := db.Create(&patient).Error; err != nil {
+		log.Warn().Err(err).Msg("database error")
+		return nil, status.Error(codes.Internal, err.Error())
+	}
+
+	log.Info().
+		Str("patientId", patient.Id.String()).
+		Msg("patient created")
+
+	return &api.CreatePatientResponse{
+		Id: patient.Id.String(),
+	}, nil
+}
+
+func (ServiceServer) GetPatient(ctx context.Context, req *api.GetPatientRequest) (*api.GetPatientResponse, error) {
+	log := zlog.Ctx(ctx)
+	db := hwgorm.GetDB(ctx)
+
+	// TODO: Auth
+
+	id, err := uuid.Parse(req.Id)
+	if err != nil {
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	patient := Patient{Id: id}
+	if err := db.First(&patient).Error; err != nil {
+		if hwgorm.IsOurFault(err) {
+			log.Warn().Err(err).Msg("database error")
```

Comment:
This is definitely worth an `Error` log level. But keep in mind, that returned errors are already logged by the [logging middleware](https://github.com/helpwave/services/blob/main/libs/common/grpc.go#L127). I'll open an issue to change the log level for interal errors from warn to error.

Reply:
Sounds good! I've removed the log there, didn't know that the logging middleware already does this.